src/sql/parts/notebook/notebookActions.ts

```diff
@@ -104,6 +105,81 @@ export abstract class ToggleableAction extends Action {
 	}
 }
 
+
+export interface IActionStateData {
+	className?: string;
```

Comment:
It's a little weird to see an interface where every property is optional, but alas, here we are :).

Reply:
Welcome to Typescript eh?